test/test_pallas.py

```diff
@@ -205,7 +205,7 @@ def test_flash_attention_wrapper(self):
 
     o = flash_attention(q, k, v)
     expected_o = self._attention(q, k, v)
-    self.assertTrue(torch.allclose(o.cpu(), expected_o.cpu()))
+    self.assertTrue(torch.allclose(o.cpu(), expected_o.cpu(), atol=1e-07))
```

Comment:
How about using 1e-4? Just in case.

Reply:
let's do 1e-5 lol, we also set the precision to highest in the test.